tests/cli/test_suite.py

```diff
@@ -27,39 +28,51 @@ def test_suite_help_output(caplog,):
     assert_no_logging_messages_or_tracebacks(caplog, result)
 
 
-def test_suite_new_on_context_with_no_datasources(caplog, empty_data_context):
+@mock.patch("subprocess.call", return_value=True, side_effect=None)
+@mock.patch("webbrowser.open", return_value=True, side_effect=None)
+def test_suite_new_on_context_with_no_datasources(
+    mock_webbrowser, mock_subprocess, caplog, empty_data_context
+):
     """
     We call the "suite new" command on a data context that has no datasources
     configured.
 
-    The command should exit with a clear error message
+    The command should:
+    - exit with a clear error message
+    - NOT open Data Docs
+    - NOT open jupyter
```

Comment:
All `suite_*`  tests updated to codify Data Docs and jupyter behavior in comments and code.